src/containers/Board/SymbolDetails/messages.js

```diff
@@ -5,4 +5,8 @@ export default defineMessages({
     id: 'cboard.containers.Board.SymbolDetails.addSymbol',
     defaultMessage: 'Add Symbol',
   },
+  editSymbol: {
+    id: 'cboard.containers.Board.SymbolDetails.editSymbol',
+    defaultMessage: 'Edit Symbol'
```

Comment:
This could be a plural "edit symbols" but since this message already existed from previous version I would stick with it. We are editing symbol one by one anyway

Reply:
for now we can leave it as is.. this can be addressed later down the road when we find some UX designer